build_tfm.py

```diff
@@ -712,6 +720,8 @@ def _main():
             return
 
     if args.clean:
+        args.skip_clone = False
```

Comment:
Or instead print an error if both are true, so the user won't be wondering why the repos are cloned?

Reply:
done.